### PR TITLE
Feature/new app version validation check

### PR DESCRIPTION
**Pull request by marcussc**
## Description
This PR introduces a new Check for TicketValidation, in which a minimum AppVersion is required to do the Ticketvalidation. If the App is outdated an error is displayed with the possibility to go to the AppStore to update the App. 
Also the Error Message and Title for SP_ALLOWLIST_NO_MATCH got updated.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10959

## Screenshots
Screen Shot 2021-12-08 at 15 33 56
Simulator Screen Shot - iPhone 13 mini - 2021-12-08 at 15 05 38

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Services/Risk/Provider/AppFeatures/AppFeatureProviding.swift [hunk 1]
```diff
@@ -8,8 +8,13 @@ extension SAP_Internal_V2_ApplicationConfigurationIOS {
 
 	/// list of known app features
 	enum AppFeature: String {
+		
 		case disableDeviceTimeCheck = "disable-device-time-check"
 		case unencryptedCheckinsEnabled = "unencrypted-checkins-enabled"
+		case isTicketValidationEnabled = "isTicketValidationEnabled"
+//		case validationServiceMinVersionMajor = "validation-service-ios-min-version-major"
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Services/Risk/Provider/AppFeatures/AppFeatureProviding.swift:15]: can these be removed?

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -3407,6 +3407,14 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "TicketValidation_Error_tryAgain" = "Es trat ein Fehler bei der Verarbeitung der Daten auf. Bitte versuchen Sie es später erneut.";
 
+"TicketValidation_Error_OutdatedApp" = "In der aktuellen Version ihrer Corona-Warn-App wird die Ticketüberprüfung nicht mehr unterstützt. Bitte aktualisieren Sie Ihre App und versuchen Sie die Ticketüberprüfung erneut.";
+
+"TicketValidation_Error_UpdateAction" = "zum App Store";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:3412]: Please start with a capital "Z" > Zum App Store

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]
Change capitalization

**Review by nickguendling — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: Done

File: src/xcode/ENA/ENA/Source/Services/Risk/Provider/AppFeatures/AppFeatureProviding.swift (hunk 1 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Services/Risk/Provider/AppFeatures/AppFeatureProviding.swift]: Done

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**